### PR TITLE
Add uptime column to show-validators

### LINKED ISSUE #7276: Improve `solana uptime` to display multiple validators
`solana uptime <PUBKEY>` allows one to easily view the uptime measurement for a given validator.  But we're missing the ability to easily view the uptime for all nodes.

Two possible solutions that I see are:
1. `solana uptime`, with no pubkey, shows the uptime for all validators
2. Augment the `solana show-validators` command to include the uptime for all validators

### PR DESCRIPTION
**Pull request by jstarry**
#### Problem
`solana uptime <PUBKEY>` allows one to easily view the uptime measurement for a given validator. But we're missing the ability to easily view the uptime for all nodes.

#### Summary of Changes
- Add a column for `Uptime` in the `show-validators` output
- Add `epoch_credits` to the `vote_accounts` RPC endpoint

Fixes #7276

### CODE REVIEW

File: client/src/rpc_request.rs [hunk 1]
```diff
@@ -97,6 +97,11 @@ pub struct RpcVoteAccountInfo {
     /// Whether this account is staked for the current epoch
     pub epoch_vote_account: bool,
 
+    /// History of how many credits earned by the end of each epoch
+    ///   each tuple is (Epoch, credits, prev_credits)
+    #[serde(default)]
```
Comment (jstarry) [inline client/src/rpc_request.rs]: Added this for backwards compatibility, do we need it?

Comment (mvines) [reply client/src/rpc_request.rs]: Backwards compatibility with what exactly?

File: cli/src/cluster_query.rs [hunk 2]
```diff
@@ -617,8 +621,26 @@ pub fn process_show_validators(rpc_client: &RpcClient, use_lamports_unit: bool)
                 format!("{}", v)
             }
         }
+
+        fn uptime(epoch_credits: Vec<(Epoch, u64, u64)>, epoch_schedule: &EpochSchedule) -> String {
```
Comment (mvines) [inline cli/src/cluster_query.rs]: The semi-duplication with `process_uptime()` doesn't feel great to me.   Think we can turn this into a common function used in both places.  That the uptime subcommand is in `vote.rs` might make this a little awkward, but the uptime subcommand could just as easily live in `cluster_query.rs`

File: client/src/rpc_request.rs (hunk 1 above)
Comment (jstarry) [reply client/src/rpc_request.rs]: When using the latest cli client against a rpc endpoint without this value present, the request fails.

Comment (mvines) [reply client/src/rpc_request.rs]: Weird, do you know why?

Comment (jstarry) [reply client/src/rpc_request.rs]: Maybe I didn't word that clearly...

With `#[serde(default)]`
```
cargo run -p solana-cli -- show-validators          
RPC Endpoint: http://beta.testnet.solana.com:8899
Active Stake: 0.001696972 SOL

  Identity Pubkey                               Vote Account Pubkey                           Commission  Last Vote  Root Block   Uptime  Active Stake
  va11wrZ2pD668e2dKXohuXiyALPxfVQjjH7zzpePavQ   7aWpg8sXjZVDLAxTohdjmmweW2RxQYWBFeBBRVLbUQN8         50%      53978       53947        -  0.000424243 SOL (25.00%)
  va13en4eUarJtf8mbhFF386nvQh12g6ESkjoR7Ji8hm   EU9SK7xG5YLpX2amTfaBuQpBFqCxvhMqxbCwbjwNb2A5         50%      53978       53947        -  0.000424243 SOL (25.00%)
  va12u4o9DipLEB2z4fuoHszroq1U9NcAB9aooFDPJSf   DTUGtGtHAdETxaUkNoDr5i9a1RgsFP9a9DoPY7cLQZpd         50%      53978       53947        -  0.000424243 SOL (25.00%)
  boot1Z6jb15CLqpaMTn2CxktktwZpRAVAgHZEW6SxQ7   3YTyUqUoZptsQHxwjxT5Yk7SBP5iL6tsWgZpn9ez4gFM          0%      53978       53947        -  0.000424243 SOL (25.00%)
```

Without `#[serde(default)]`
```
cargo run -p solana-cli -- show-validators
RPC Endpoint: http://beta.testnet.solana.com:8899
Error: Custom { kind: Other, error: "GetVoteAccounts parse failure: missing field `epochCredits`" }
```

File: cli/src/cluster_query.rs (hunk 2 above)
Comment (jstarry) [reply cli/src/cluster_query.rs]: Sounds good, added a method here

File: client/src/rpc_request.rs (hunk 1 above)
Comment (mvines) [reply client/src/rpc_request.rs]: Oh I see, you're mixing master and v0.21.  I don't think we need this.  Once this PR lands on v0.21 this situation will resolve itself.

**Pull request merged**